build.sbt

```diff
@@ -3,11 +3,11 @@ import sbt.Keys.parallelExecution
 import sbt.{Compile, moduleFilter, _}
 import sbtassembly.AssemblyPlugin.autoImport.assembly
 
-lazy val scala212 = "2.12.11"
-lazy val supportedScalaVersions = List(scala212)
+lazy val scala213 = "2.13.8"
+lazy val supportedScalaVersions = List(scala213)
```

Comment:
We would like to support 2.12 and 2.13. Could you please adjust the build to support both versions?
Additionally, could you modify github actions definition to run for 2.13 as well?

Reply:
Done, I rebased my change on top of master and added 2.12 to here and GitHub actions.

I am not a Scala developer normally, nor do I work with Github actions, so I have no idea if I did it correctly, so apologies if it doesn't actually work. `sbt test` seems to pass for me.

Reply:
Thanks! We definitely need https://github.com/scala/scala-collection-compat to make this work as the adjusted code  isn't compatible with 2.12. You  may try it yourself by typing `sbt/sbt ++2.12.11 test` in your cmd.

Unfortunately even with scala-collection-compat we have problems with `scala.collection.mutable.Builder` used `om.datastax.spark.connector.types.CanBuildFrom`. The api is different, we might want to substitute the builder with a custom class. Would you like to take a look? If not I will give a try later.

Reply:
scala compat: https://github.com/marquiswang/spark-cassandra-connector/pull/1

Reply:
Many other Spark libraries release two binaries for the different Scala versions. Maybe we just want to do that?

https://search.maven.org/search?q=a:spark-launcher_2.1*

Reply:
Yeah, that's the goal here. We want to keep a single code base tho.

Reply:
Ah, I had assumed those projects just have a few slightly different branches.

I'm not sure if I'm going to be able to deal with the compatibility issues. Probably better for you to have a look.

Reply:
Could you please sign https://cla.datastax.com/ ?

Reply:
I believe I've already signed it! It shows as "Completed".

Reply:
I'll work on Scala 2.13 support in the background and in the meantime I'll publish Spark 3.3 with Scala 2.12 support. Thank you for this contribution!